vent/menus/add.py

```diff
@@ -16,7 +16,7 @@ def create(self):
         self.repo.when_value_edited()
 
     def quit(self, *args, **kwargs):
-        self.parentApp.switchForm(None)
+        self.parentApp.switchForm('MAIN')
```

Comment:
yeah, i think it's just that one repo_value variable that I've been using so far.  so all that needs to happen is for it to get set back to `{}` on cancel or completion of the add plugin process (those 3 menus).